src/course/model.coffee

```diff
@@ -64,6 +64,15 @@ class Course
     else
       _.first(names)
 
+  getStudentIdentifier: ->
+    @getStudentRecord()?.student_identifier
+
+  getStudentRecord: ->
+    # Currently the students listing only contains the current student
+    # If that is ever extended then the bootstrap data will need to include
+    # the current user's id so that the `students` array can be searched for it.
+    _.first(@students) or @students ||= [{}]
```

Comment:
If students is completely missing, as it is after a fresh registration then the other side is evaluated.   But yeah your correct , it could/should just be an equals. I'll correct it